### PR TITLE
parse_ply

### PR DESCRIPTION
**Pull request by tyamagis**
parse_ply.c
parse_ply.h

### minimal parse from [.ply] to [t_ply].
 - NOT contains file format check.
 - Contains a wasteful-looking process. (will be changed.)
 - parse_ply.c contains main() to test comment-outed.

### CODE REVIEW

File: parse_ply.c [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	i = 0;
+	while (i++ < ply->elem_vertex)
+	{
+		fscanf(f_stream, "%f %f %f", &vtx->x, &vtx->y, &vtx->z);
+		vtx++;
+	}
+	return (ply->vertexes);
+}
+
+t_ply	*set_elem(char *filename)
```
Comment (velvelhype) [inline parse_ply.c:90]: set_elem is ambiguous name.
I like parse_ply.

File: parse_ply.c [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+/*   By: tyamagis <tyamagis@student.42tokyo.>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/02/16 14:56:58 by tyamagis          #+#    #+#             */
+/*   Updated: 2022/02/16 18:26:32 by tyamagis         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "./include/ply.h"
+
+void	*poly_to_tri(t_ply *ply, int n, FILE **f_stream)
```
Comment (velvelhype) [inline parse_ply.c:15]: "ply" is too abstract for me.
"ply_info" is more stress free for me.

File: include/parse_ply.h [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+typedef struct s_vertex
+{
+	float	x;
+	float	y;
+	float	z;
+} t_vertex;
+
+typedef struct s_ply
+{
+	int			elem_vertex;
```
Comment (velvelhype) [inline include/parse_ply.h:37]: elem_vertex -> elem_vertexes
elem_face -> elem_faces

**Review by velvelhype — COMMENTED** [review]
checked.

File: include/parse_ply.h (hunk 3 above)
Comment (tyamagis) [reply include/parse_ply.h:37]: Done

File: parse_ply.c (hunk 2 above)
Comment (tyamagis) [reply parse_ply.c:15]: DONE

File: parse_ply.c (hunk 1 above)
Comment (tyamagis) [reply parse_ply.c:90]: DONE

**Pull request merged**